src/Helpers/MiscHelper.js

```diff
@@ -159,9 +160,11 @@ export const useUrlParams = (allowedParams) => {
     const url = new URL(window.location);
     const urlParams = qs.parse(url.search);
```

Comment:
Can `useLocation` maybe replace these custom consts?

Reply:
Yeah, I replaced it with the top one,
the second one could be replaced with
```js
const urlParams = Object.fromEntries(new URLSearchParams(location.search));
```
but the `qs` package takes care of the params when there are multiple params with the same name, so I would keep the `qs` there.